src/main/battlecode/common/RobotController.java

```diff
@@ -1125,6 +1134,14 @@
     // ***********************************
 
     /**
+     * Returns the current cost of a victory point in bullets. This varies based
+     * on the round number, and is equal to 10 + RoundNum/300

```

Comment:
This is inconsistent with what was announced on stream and is suggested in common.GameConstants